package.json

```diff
@@ -3,11 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.2.25",
-    "@fortawesome/free-solid-svg-icons": "^5.11.2",
-    "@fortawesome/react-fontawesome": "^0.1.7",
-    "aws-sdk": "^2.586.0",
-    "axios": "^0.19.0",
+    "aws-sdk": "^2.570.0",
```

Comment:
Why did the minimum version drop? I'm sure it's fine, just curious.

Reply:
Merge conflict with dependent update